khelo/lib/ui/flow/my_game/my_game_tab_screen.dart

```diff
@@ -126,16 +126,15 @@ class _MyGameTabScreenState extends ConsumerState<MyGameTabScreen>
           ),
           if (_selectedTab == 1) ...[
             const Spacer(),
-            actionButton(context, onPressed: () {
-              ref.read(teamListViewStateProvider.notifier).onFilterButtonTap();
-            }, icon: const Icon(CupertinoIcons.slider_horizontal_3)),
-            actionButton(context, onPressed: () {
-              AppRoute.addTeam().push(context);
-            },
-                icon: Icon(
-                  Icons.add,
-                  color: context.colorScheme.primary,
-                )),
+            actionButton(context,
+                onPressed: () => ref
+                    .read(teamListViewStateProvider.notifier)
+                    .onFilterButtonTap(),
+                icon: Icon(CupertinoIcons.slider_horizontal_3,
+                    color: context.colorScheme.textPrimary)),
```

Comment:
even though in figma it's **textPrimary**, can these button's color be **primary**? we keep action buttons primary and this screen doesn't have appBar but these buttons are like action button.